cmd/kube-apiserver/app/server.go

```diff
@@ -123,7 +125,7 @@ func Run(s *options.ServerRunOptions) error {
 
 	// Setup tunneler if needed
 	var tunneler genericapiserver.Tunneler
-	var proxyDialerFn apiserver.ProxyDialerFunc
+	var proxyDialerFn func(network, addr string) (net.Conn, error)
```

Comment:
This looks like a move in the wrong direction.  There ought to be *some* type we can use.